components/post_view/post/post.jsx

```diff
@@ -265,7 +265,7 @@ export default class Post extends React.PureComponent {
                 className={this.getClassName(post, isSystemMessage, fromWebhook, fromAutoResponder)}
                 onMouseOver={this.setHover}
                 onMouseLeave={this.unsetHover}
-                onTouchStart={this.setHover}
+                onPointerOver={this.setHover}
```

Comment:
This might not work on iOS. According to the MDN, it doesn't support Safari ([link](https://developer.mozilla.org/en-US/docs/Web/Events/pointerover))

Reply:
@hmhealey Good catch. I tested this on chrome with simulators
This PR is not needed as the ticket was opened by testing on pre-release but i didn't realise fix is not on pre-release yet. 

Had katie test on ci server and earlier PR works look. Closing this to wait for rc2 to test on pre-release.